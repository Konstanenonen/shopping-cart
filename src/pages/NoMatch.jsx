import React from 'react';
import styles from '../styles/pages/NoMatch.module.scss';

function NoMatch() {
  return <h1 className={styles.text}>Ooops! There is nothing here.</h1>;
}

export default NoMatch;
