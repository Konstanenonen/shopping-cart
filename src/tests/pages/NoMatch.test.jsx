/* global it describe expect */

import React from 'react';
import { render } from '@testing-library/react';
import NoMatch from '../../pages/NoMatch';

describe('NoMatch page', () => {
  it('renders "Ooops! There is nothing here."', () => {
    const { container } = render(<NoMatch />);
    expect(container).toMatchSnapshot();
  });
});
