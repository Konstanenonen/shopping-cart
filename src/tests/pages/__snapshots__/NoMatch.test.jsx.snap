// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoMatch page renders "Ooops! There is nothing here." 1`] = `
<div>
  <h1
    class="text"
  >
    Ooops! There is nothing here.
  </h1>
</div>
`;
